apps/web/src/components/BlocksEditor/Editor/state/promptlToLexical/index.test.ts

```diff
@@ -46,6 +46,250 @@ pong
+        type: 'paragraph',
+        children: [
+          expect.objectContaining({
+            type: 'text',
+            text: '',
+          }),
+        ],
+      }),
+      expect.objectContaining({
+        type: 'step',
+        children: [
+          expect.objectContaining({
+            type: 'paragraph',
+            children: [
+              expect.objectContaining({
+                type: 'text',
+                text: 'This is a system message',
+              }),
+            ],
+          }),
+          expect.objectContaining({
+            type: 'paragraph',
+            children: [
+              expect.objectContaining({
+                type: 'text',
+                text: '',
+              }),
+            ],
+          }),
+          expect.objectContaining({
+            type: 'message',
+            role: 'user',
+            children: [
+              expect.objectContaining({
+                type: 'paragraph',
+                children: [
+                  expect.objectContaining({
+                    type: 'text',
+                    text: 'This is a user message',
+                  }),
+                ],
+              }),
+            ],
+          }),
+          expect.objectContaining({
+            type: 'paragraph',
+            children: [
+              expect.objectContaining({
+                type: 'text',
+                text: '',
+              }),
+            ],
+          }),
+          expect.objectContaining({
+            type: 'paragraph',
+            children: [
+              expect.objectContaining({
+                type: 'text',
+                text: 'This is a list:',
+              }),
+            ],
+          }),
+          expect.objectContaining({
+            type: 'paragraph',
+            children: [
+              expect.objectContaining({
+                type: 'text',
+                text: '  - This is a list item',
+              }),
+            ],
+          }),
+          expect.objectContaining({
+            type: 'paragraph',
+            children: [
+              expect.objectContaining({
+                type: 'text',
+                text: '  - This is a list item',
+              }),
+            ],
+          }),
+          expect.objectContaining({
+            type: 'paragraph',
+            children: [
+              expect.objectContaining({
+                type: 'text',
+                text: '',
+              }),
+            ],
+          }),
+          expect.objectContaining({
+            type: 'message',
+            role: 'system',
+            children: [
+              expect.objectContaining({
+                type: 'paragraph',
+                children: [
+                  expect.objectContaining({
+                    type: 'text',
+                    text: 'This is an inline system message',
+                  }),
+                ],
+              }),
+            ],
+          }),
+        ],
+      }),
+    ]
```

Comment:
When I worked on this, I also did tests expecting blocks, but in reality, I think checking the output is just fine